db/migrate/20170920193238_add_employee.rb

```diff
@@ -6,6 +6,8 @@ def change
       t.string :email
       t.string :phone
 
+      t.belongs_to :office
+
```

Comment:
Didn't these first two migration files already migrate? Migrations that have already run should basically never be edited.

The timestamp in the filename is used to check if the migration needs to be run for a specific database. There's a table in every standard Rails database with one column called `schema_migrations` and each row contains one datestamp as a string. Whenever the database is migrated, migrations that use a timestamp that is already in that table will be skipped.

So: Create a new migration or move these edits into the `add_office` migration. You'll need to use `add_belongs_to` or `add_reference` outside of the table context that's available here.

Reply:
@ryannoort: Remember that this needs to be fixed. This is the only thing holding me back from accepting. There's also the change that Zach recommended on `EmployeesController` which has some value but isn't a blocker in my mind.